Velaptor/Services/FontBatchService.cs

```diff
@@ -0,0 +1,124 @@
+﻿// <copyright file="FontBatchService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Services
+{
+    // ReSharper disable RedundantNameQualifier
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using Velaptor.OpenGL;
+
+    // ReSharper restore RedundantNameQualifier
+
+    /// <summary>
+    /// Manages the process of batching up the rendering of text.
+    /// </summary>
+    internal class FontBatchService : IBatchManagerService<FontBatchItem>
+    {
+        private SortedDictionary<uint, (bool shouldRender, FontBatchItem item)> batchItems = new ();
+        private uint currentBatchIndex;
+        private uint batchSize;
+        private uint currentFrame;
+
+        /// <summary>
+        /// Occurs when a batch is full.
+        /// </summary>
+        /// <remarks>
```

Comment:
Refactor remarks section to only include:

`The batch is ready when the total amount of items to be rendered is equal to the <see cref="BatchSize"/>.`